core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.component.ts

```diff
@@ -512,6 +569,22 @@ angular.module('oppia').component('stateTranslation', {
           StateEditorService.onRefreshStateTranslation.subscribe(
             () => $scope.initStateTranslation())
         );
+        ctrl.directiveSubscriptions.add(
+          TranslationTabActiveModeService
```

Comment:
Why do we need this now, when we did not before?

The justification in the comment below seems odd to me. The rule inputs tab should simply not appear in voiceover mode.